compiler/src/iree/compiler/Codegen/SPIRV/SPIRVTileAndVectorizeToCooperativeOps.cpp

```diff
@@ -64,6 +72,35 @@ static SmallVector<int64_t> getTargetCooperativeOpSize(linalg::LinalgOp op) {
   return getTileSizes(op, 3); // For native vector sizes
 }
 
+static constexpr char coopMatShapeAttrName[] = "iree.spirv.coop_mat_shape";
```

Comment:
nit: `static` is redundant for global constants
```suggestion
constexpr char coopMatShapeAttrName[] = "iree.spirv.coop_mat_shape";
```